types/ltx/src/Parser.d.ts

```diff
@@ -1,5 +1,3 @@
-/// <reference types="node" />
```

Comment:
so shoudnt that go to `index.d.ts`?

Reply:
I wanted add it on where it is actually needed.